enhancements/insights/rapid-recommendations.md

```diff
@@ -0,0 +1,552 @@
+  for a future extension of the remote configuration feature.
+* **Remote configuration for collecting Prometheus metrics and alerts.** This feature will be considered 
+  as a future extension of the remote configuration feature.
+* **Remote configuration for collecting data about layered products.** Layered products are often 
+  deployed into custom namespaces which poses unique challenges. 
+  We will explore options to extend this feature to layered products later.
+* **Replacing all existing gatherers.** Data that require advanced processing by the Insights Operator
+  (e.g. workload fingerprints) will keep being collected by hardcoded Insights Operator components. 
+  There are no plans to change this.
+* **Air-gapped and disconnected clusters.** While the proposed solution enables manual workarounds, 
+  we do not think they should be presented as intended solutions for these use cases. 
+  These use cases should be addressed separately.
+
+ 
+## Proposal
+
+The proposal is based on a concept already used by the 
+[Conditional data gathering](../insights/conditional-data-gathering.md) feature of the Insights Operator:
+Before the Insights Operator starts creating a new archive (i.e. every 2 hours), 
+it will download remote configuration from a service on console.redhat.com 
+and gather data specified in the configuration. 
+The Observability Intelligence team (former CCX) will maintain the remote configuration in a repository 
+and deploy it to console.redhat.com whenever it changes.
+
+### Workflow Description
+
+This proposal introduces a new workflow for requesting new data to be collected by the Insights Operator.
+
+A **data requester** is a person who analyzes or develops tools to analyze Insights Operator archives
+uploaded to Red Hat.
+
+1. A data requester identifies the need for new data in the Insights Operator archive.
+2. The data requester describes the data using a format required by the remote configuration service.
+3. The data requester creates a pull request to the repository where the remote configuration is maintained.
+4. The pull request is reviewed (and either approved or denied) 
+   by a member of the Observability Intelligence (former CCX) team.
+5. When the pull request is merged, the new remote configuration is deployed 
+   to the remote configuration service on console.redhat.com.
+6. Insights Operators in connected clusters download the new remote configuration 
+   and start collecting the new data.
+7. The data requester makes use of the new data in newly incoming Insights Operator archives.
+
+### API Extensions
+
+This proposal does not add any API extension.
+
+### Topology Considerations
+
+#### Hypershift / Hosted Control Planes
+
+This proposal does not require any specific details to account for the HyperShift. 
+For now, we can simply extend the remote configuration file to 
+request gathering of specific HyperShift resources.
```

Comment:
How does this work, if you identify the `(namespace, pod, container)`  you want logs from, but on HyperShift that controller happens to live on the management cluster?  Where in the pipe does the request get shared between where-that-controller-lives-in-standalone vs. where-that-controller-lives-in-hosted?

Reply:
So the Insights Operator is installed on both - management and worker cluster. I think the idea is simply, if the data is available in the cluster, then collect it, otherwise provide information (in the archive) about why it was not collected.